### PR TITLE
[TECH-2248] Session timeout error screen

### PR DESCRIPTION
**Pull request by av-alexistoledo**
* Plus `RUNNING OUT OF TIME` alert.

https://github.com/aion-dk/us-dbas/assets/112619860/a9c0c517-0836-4fec-84d5-3ba0e56abbe8

https://github.com/aion-dk/us-dbas/assets/112619860/1ce2f572-0c87-4bfe-a198-925ab25913b3

### CODE REVIEW

File: client/src/components/Timedown.vue [hunk 1]
```diff
@@ -20,22 +20,33 @@ function decrementSeconds() {
...
 );
 const style = computed(() => {
   return `
       --width: ${percentageWidth.value}%;
+      --progress-bg: ${
+        secondsLeft.value <= ALERT_TIMEOUT ? "#ee9500" : "#09132f"
+      } 
     `;
+  // #ee9500 -> var(--semantic-warning-1)
+  // #09132f -> var(--slate-800)
```
Comment (av-martin) [inline client/src/components/Timedown.vue:43]: Are the variables not available?

File: client/src/views/BallotTrackerStart.vue [hunk 2]
```diff
@@ -148,8 +148,7 @@ onMounted(() => {
 
 <style type="text/css" scoped>
 .BallotTrackerStart {
-  font-family: "Open Sans";
```
Comment (av-martin) [inline client/src/views/BallotTrackerStart.vue:151]: Was the font not supposed to be "Open Sans", or is it set elsewhere?

File: client/src/components/Timedown.vue [hunk 3]
```diff
@@ -20,22 +20,33 @@ function decrementSeconds() {
...
+  let min = Math.floor(ms / 60000);
+  let sec = Number.parseInt(((ms % 60000) / 1000).toFixed(0));
+  return min + ":" + (sec < 10 ? "0" : "") + sec;
+};
+
 const secondsLeft = ref(props.currentSeconds);
 const interval = ref(null);
 const miliseconds = computed(() => secondsLeft.value * 1000);
-const displayValue = computed(() =>
-  humanizeDuration(miliseconds.value, { language: i18n.global.locale })
```
Comment (av-martin) [inline client/src/components/Timedown.vue:27]: Did this not work as required?

File: client/src/components/Timedown.vue (hunk 1 above)
Comment (av-alexistoledo) [reply client/src/components/Timedown.vue:43]: Yes, I found a way to refactor it 😅

File: client/src/views/BallotTrackerStart.vue (hunk 2 above)
Comment (av-alexistoledo) [reply client/src/views/BallotTrackerStart.vue:151]: It is now globally declared for the whole `body` so there's no need to have it in each class. I'm cleaning those up when I found them throughout the components I'm currently working on.

File: client/src/components/Timedown.vue (hunk 3 above)
Comment (av-alexistoledo) [reply client/src/components/Timedown.vue:27]: It works as it is supposed but it doesn't have any configuration for what we need. The module is thought to give you a detailed description of the time (1 hour, 20 minutes and 5 seconds + localizations) but according to the prototypes we just need a `mm:ss`, without any text or translations.

https://www.npmjs.com/package/humanize-duration

**Review by av-martin — APPROVED** [review]

**Pull request merged**